packages/oui-header-tabs/header-tabs.less

```diff
@@ -0,0 +1,79 @@
+.oui-tabs {
+  list-style: none;
+  display: flex;
+  padding: 0;
+  margin: 0;
+  background-color: @oui-color-nurse;
+  height: rem-calc(44px);
+
+  &__tab {
+    position: relative;
+    padding: 0;
+    font-weight: 400;
+    transition: all 1000ms ease-in;
+
+    a {
+      display: block;
+      padding: rem-calc(11px) rem-calc(24px);
+      color: @oui-color-sapphire;
+      text-decoration: none;
+      font-family: SourceSansPro-Regular;
```

Comment:
There is mixins for that, check oui-typography. 
Or use `@oui-global-base-font-family` if you only need the font-family. The property value is not right.